### PR TITLE
Release 7.2.0

### PR DESCRIPTION
**Pull request by rlepinski**
Still need to verify the changes

### CODE REVIEW

File: ios/Classes/SwiftAirshipPlugin.swift [hunk 1]
```diff
@@ -18,37 +18,53 @@ public class SwiftAirshipPlugin: NSObject, FlutterPlugin {
         .notificationStatusChanged: "com.airship.flutter/event/notification_status_changed"
     ]
 
-    private static let streams: [AirshipProxyEventType: AirshipEventStream] = {
+
+    private static let eventSubject = PassthroughSubject<AirshipProxyEventType, Never>()
+    private static let eventTask: Task<Void, Never> =  Task {
```
Comment (rlepinski) [inline ios/Classes/SwiftAirshipPlugin.swift:23]: Uses a task to send to a subject so we can have multiple listeners

File: ios/Classes/SwiftAirshipPlugin.swift [hunk 2]
```diff
@@ -18,37 +18,53 @@ public class SwiftAirshipPlugin: NSObject, FlutterPlugin {
...
+        let pending = await AirshipProxyEventEmitter.shared.pendingEventAdded
+        for await event in pending {
+            eventSubject.send(event.type)
+        }
+    }
+
+    private let streams: [AirshipProxyEventType: AirshipEventStream] = {
```
Comment (rlepinski) [inline ios/Classes/SwiftAirshipPlugin.swift:30]: Made these a local variable so we can listen from multiple sources

**Comment by crow** [comment]
Static constants are lazily initialized on their first access. It looks like both eventTask and eventSubject are still uninitialized on their first use which causes pending events processing to not occur for incoming events like push received, message center displays, etc. Initializing eventTask and eventSubject to a discardable result on init (or on register...) seems to resolve this issue. Will push a fix when I have access.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Flutter Plugin Changelog
 
+## Version 7.2.0 - November 27, 2023
+Minor release that improves Feature Flag support on iOS.
+
+### Changes
+- Updated Airship iOS SDK to 17.6.1.
```
Comment (rlepinski) [inline CHANGELOG.md:7]: Updated Airship Android SDK to 17.5.0

File: ios/Classes/SwiftAirshipPlugin.swift [hunk 4]
```diff
@@ -18,37 +18,53 @@ public class SwiftAirshipPlugin: NSObject, FlutterPlugin {
         .notificationStatusChanged: "com.airship.flutter/event/notification_status_changed"
     ]
 
-    private static let streams: [AirshipProxyEventType: AirshipEventStream] = {
+
+    private static let eventSubject = PassthroughSubject<AirshipProxyEventType, Never>()
+    private static let eventTask: Task<Void, Never> =  Task {
+        let pending = await AirshipProxyEventEmitter.shared.pendingEventAdded
+        for await event in pending {
+            eventSubject.send(event.type)
```
Comment (rlepinski) [inline ios/Classes/SwiftAirshipPlugin.swift:26]: We should be able to remove this now that `AirshipProxyEventEmitter` exposes a subject for pending

Comment (rlepinski) [reply ios/Classes/SwiftAirshipPlugin.swift]: You need to remove the publisher as well here and then have the streams use AIrshipProxyEventEmitter.shared.eventSubject (i should of exposed a publisher but oh well, its internal).

**Comment by crow** [comment]
waiting on the latest airship framework proxy release (5.0.2) before re-running tests

**Review by crow — APPROVED** [review]

**Review by jyaganeh — APPROVED** [review]

**Pull request merged**